uint8 brake
uint8 alarm
uint8 ready
uint8 zspd
uint8 inpos1
uint8 tlmt
uint8 vlmt
uint8 inspd
uint8 warn
uint8 tgon
uint8 reserved1
uint8 reserved2
uint8 reserved3
uint8 reserved4
uint8 reserved5
uint8 reserved6
uint8 org
uint8 eos
uint8 iout0
uint8 iout1
uint8 iout2
uint8 iout3
uint8 iout4
uint8 iout5
uint8 reserved7
uint8 reserved8
uint8 reserved9
uint8 reserved10
uint8 reserved11
uint8 reserved12
uint8 reserved13
uint8 reserved14